services/cirrus/cirrus.service.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  message: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('unknown'))
+    .required(),
+}).required()
+const queryParamSchema = Joi.object({
+  task: Joi.string(),
+  script: Joi.string(),
+}).required()
+
+module.exports = class Cirrus extends BaseJsonService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'cirrus',
+      pattern: '/github/:user/:repo/:branch*',
+      queryParamSchema,
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Cirrus CI - Base Branch Build Status',
+        namedParams: { user: 'flutter', repo: 'flutter' },
+        staticPreview: this.constructor.render({ subject: 'Ci', status: 'passing' }),
```

Comment:
Another bad suggestion from me (problem with debugging by eyeballing alone 😉 )
```suggestion
        pattern: '/github/:user/:repo',
        queryParams: { task: 'analyze', script: 'test' },
        staticPreview: this.render({ subject: 'Ci', status: 'passing' }),
```